samza-core/src/main/java/org/apache/samza/runtime/RemoteApplicationRunner.java

```diff
@@ -35,12 +37,32 @@
  */
 public class RemoteApplicationRunner extends AbstractApplicationRunner {
 
-  private static final Logger log = LoggerFactory.getLogger(RemoteApplicationRunner.class);
+  private static final Logger LOG = LoggerFactory.getLogger(RemoteApplicationRunner.class);
```

Comment:
@xinyuiscool I prefer it to be in the abstract class itself as it is much cleaner. It is pretty common for interfaces to throw exception when an operation is not supported. Any particular reason you want to keep it only in LocalApplicationRunner?